cristin-commons/src/main/java/no/unit/nva/model/Organization.java

```diff
@@ -167,11 +170,25 @@ public Builder withAcronym(String acronym) {
         }
 
         public Builder withPartOf(Set<Organization> partOf) {
+            if (nonNull(partOf)) {
+                this.partOf = new ArrayList<>(partOf);
+            }
+            return this;
+        }
+
+        public Builder withPartOf(List<Organization> partOf) {
```

Comment:
You may be able to avoid this by using `withPartOf(Collection<Organization> partOf)` and the method above.